docs/website/docs/dlt-ecosystem/destinations/databricks.md

```diff
@@ -309,14 +309,41 @@ print(pipeline.dataset().pokemon.df())
 - **For production**, explicitly setting *staging_volume_name* is recommended.
 - The volume is used as a **temporary location** to store files before loading.
 
-:::tip::
+:::tip
 You can delete staged files **immediately** after loading by setting the following config option:
 ```toml
 [destination.databricks]
 keep_staged_files = false
 ```
 :::
 
+## Supported hints
+
+### Supported table hints
+
+Databricks supports the following table hints:
+
+- `description` - Uses the description to add comment to the table. This can also be done by using the adapter method `table_comment`.
```

Comment:
> by using the adapter method `table_comment`.
@bayees could you clarify if you mean "adapter method  `table_comment`" or "adapter parameter `table_comment`"?